cpp/src/gandiva/CMakeLists.txt

```diff
@@ -154,8 +154,7 @@ install(FILES
   selection_vector.h
   status.h
   tree_expr_builder.h
-  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
-)
+  DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/gandiva")
```

Comment:
Because it is possible to include `<gandiva/*.h>`  in gandiva-glib.
But I’m not sure whether this change is correct.

Reply:
Looks right to me

Reply:
It's correct.

Reply:
Thank you for checking.